en/contribute/unit_tests.md

```diff
@@ -1,7 +1,17 @@
 # Unit Tests
 
-QGC contains a set of unit tests which must pass before a change will be considered through a pull request. The addition of new complex subsystems to QGC should include a corresponding new unit test to test it.
+*QGroundControl* (QGC) contains a set of unit tests that must pass before before a pull request will be accepted. The addition of new complex subsystems to QGC should include a corresponding new unit test to test it.
 
-You can run unit tests from the command line using the ```--unittest``` command line option which will runs all tests. You can run a specific unit test by specifying it: ```--unittest:RadioConfigTest```.
+The full list of unit tests can be found in [UnitTestList.cc](https://github.com/mavlink/qgroundcontrol/blob/master/src/qgcunittest/UnitTestList.cc).
 
-The full list of unit tests can be found in UnitTestList.cc.
\ No newline at end of file
+To run unit tests:
+1. Build in `debug` mode with `UNITTEST_BUILD` definition. 
+1. Copy the **deploy/qgroundcontrol-start.sh** script in the **debug** directory
+1. Run *all* unit tests from the command line using the `--unittest` command line option:
+   ```
+   qgroundcontrol-start.sh --unittests
```

Comment:
Is it `unittest` or `unittests` ?

Reply:
> IT is --unittest.

Thanks. Fixed.

>  Also the explanation of you to run unit tests should not be OS specific.

@DonLakeFlyer Can you explain where I have made the explanation OS specific? As far as I can tell this is all completely OS agnostic.

Reply:
Is it possible to run this unit tests from GitLab or GitHub account??
It have to be the same application or atand alone just for tests?
Someone have yml-script to run tests in github/gitlab??

Reply:
@NickNick1 Suggest you ask on QGC discussion channels. This issue is long closed.